lang/en/cms.php

```diff
@@ -103,6 +103,12 @@
 $string['error:no_instance_hash'] = 'Module {$a} has no instance hash.';
 $string['error:no_config_hash'] = 'Module {$a} has no config hash.';
 
+// Search strings.
+$string['search:cmsfield'] = 'CMS';
+$string['search:settings'] = 'Search settings';
```

Comment:
Looks like those strings are not used anymore.

Reply:
It was used for search area settings, but the settings have been removed.
I just removed this lang string.